src/utilities/permissions.ts

```diff
@@ -513,8 +513,8 @@ const queryPermissions = {
   DELETE_PARCEL: (user: User | null, parcel: AssetWithOwner<Parcel>): boolean => {
     return isUserAdmin(user) || (getPermission([Queries.DELETE_PARCEL], user) && isUserOwner(user, parcel));
   },
-  DELETE_PARCEL_TO_PARAMETER_DICTIONARIES: (user: User | null): boolean => {
-    return isUserAdmin(user) || getPermission([Queries.DELETE_PARCEL_TO_PARAMETER_DICTIONARY], user);
+  DELETE_PARCEL_TO_DICTIONARY_ASSOCIATION: (user: User | null): boolean => {
```

Comment:
I'm seeing that I no longer have permission to delete anything from the tables within the parcels page whereas on develop I can. Is this intended?
<img width="375" alt="image" src="https://github.com/NASA-AMMOS/aerie-ui/assets/4419607/da7180fc-2607-43cf-8800-3e92c0087607">

Reply:
Which role are you logged in as? This should be admin only.

Reply:
This is as admin role